tool/tctl/common/token_command.go

```diff
@@ -42,6 +43,19 @@ import (
 	"github.com/gravitational/teleport/lib/utils"
 )
 
+var mdmTokenAddTemplate = template.Must(
+	template.New("mdmTokenAdd").Parse(`The invite token: {{.token}}
+This token will expire in {{.minutes}} minutes.
+
+Use this token to add an MDM service to Teleport.
+
+> teleport start \
+   --token={{.token}} \{{range .ca_pins}}
+   --ca-pin={{.}} \{{end}}
+   -c=/path/to/mdm_service.yaml
```

Comment:
```suggestion
   -c=/path/to/teleport.yaml
```

Thus far we have always referred to the config as teleport.yaml, no matter what services are enabled in there. I don't see any reason to deviate here, we might just confuse people.

Reply:
Seems fair to me, let me change it.